From 5100551511ef2c6f6739a0987ab5d871efdc09d3 Mon Sep 17 00:00:00 2001
From: PJBrs <pjbrs@floorenpj.xs4all.nl>
Date: Sun, 16 Feb 2020 16:36:13 +0100
Subject: [PATCH] ARM: dts: sunxi: h2-plus-bananapi-m2-zero: Add ethernet

This patch enables the ethernet driver for the BananaPi M2 Zero
---
 arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts b/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
index d277d043031b..af001b12ea8f 100644
--- a/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
+++ b/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
@@ -151,3 +151,11 @@
 	 */
 	status = "okay";
 };
+
+
+&emac {
+	status = "okay";
+	phy-handle = <&int_mii_phy>;
+	phy-mode = "mii";
+	allwinner,leds-active-low;
+};
